docs/sources/operator/operator-integrations.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Set up Agent Operator integrations
+weight: 350
+---
+# Set up Agent Operator integrations
+
+This topic provides examples of setting up Agent Operator integrations, including [node_exporter](#set-up-an-agent-operator-node_exporter-integration) and [mysqld_exporter](#set-up-an-agent-operator-mysqld_exporter-integration).
+
+## Before you begin
+
+Before you begin, make sure that you have deployed the Grafana Agent Operator CRDs and installed Agent Operator into your cluster. See [Install Grafana Agent Operator with Helm]({{< relref "./helm-getting-started.md" >}}) or [Install Grafana Agent Operator]({{< relref "./getting-started.md" >}}) for instructions.
+
+Also, make sure that you [deploy the GrafanaAgent resource]({{< relref "./deploy-agent-operator-resources.md" >}}) and the `yaml` you use has the `integrations` definition under `spec`.
+
+## Set up an Agent Operator node_exporter integration
+
+The Agent Operator node_exporter integration lets you monitor your hardware and OS metrics from Unix-based machines, including Linux machines.
+
+To set up a node_exporter integration:
+
+1. Copy the following manifest to a file:  
+
+    ```yaml
+    # Collect node_exporter metrics
+    apiVersion: monitoring.grafana.com/v1alpha1
+    kind: Integration
+    metadata:
+     name: node-exporter
+     namespace: default
+     labels:
+       agent: grafana-agent-integrations
+    spec:
+     name: node_exporter
+     type:
+       allNodes: true
+       unique: true
+     config:
+       autoscrape:
+         enable: true
+         metrics_instance: default/primary
+       rootfs_path: /default/node_exporter/rootfs
+       sysfs_path: /default/node_exporter/sys
+       procfs_path: /default/node_exporter/proc
+     volumeMounts:
+       - mountPath: /default/node_exporter/proc
+         name: proc
+       - mountPath: /default/node_exporter/sys
+         name: sys
+       - mountPath: /default/node_exporter/rootfs
+         name: root
+     volumes:
+       - name: proc
+         hostPath:
+           path: /proc
+       - name: sys
+         hostPath:
+           path: /sys
+       - name: root
+         hostPath:
+           path: /root
+    ```
+
+    > **Important:** In order to make integrations work, the field `name` under the `spec` section of the manifest must contain the name of the integration to be installed according to the list of integrations defined [here]({{< relref "../configuration/integrations/integrations-next/_index.md#config-changes" >}}).
```

Comment:
@marctc I moved this paragraph from the overview to within the steps and made it an "important" note.  I think that works better to include the information right when they need it.